docs/webhooks.rst

```diff
@@ -90,7 +89,7 @@ Using the generic API integration
 For repositories that are not hosted with a supported provider, we also offer a
 generic API endpoint for triggering project builds. Similar to webhook
 integrations, this integration has a specific URL, found on the project's
-**Integrations** Admin dashboard page on readthedocs.org.
+:guilabel:`Admin` > :guilabel:`Integrations` dashboard page on readthedocs.org.
```

Comment:
I think we need to modify the above sentence `...found on the project's`

Reply:
:guilabel:`Admin` > :guilabel:`Integrations` found on the project's dashboard page.

Reply:
Similar to webhook integrations,
this integration has a specific URL, which can be found on the project's Integrations dashboard page.
(:guilabel:`Admin` > :guilabel:`Integrations`).